ament_package/template/prefix_level/_local_setup_util.py

```diff
@@ -67,6 +73,8 @@ def main(argv=sys.argv[1:]):  # noqa: D103
             args.additional_extension
         ):
             print(line)
+        for line in _remove_trailing_separators():
+            print(line)
```

Comment:
This change regresses the time to source the setup file from 0.17s to 3.67s for me with the ROS Eloquent (plus a few more packages) installed from Debian packages.

The cleanup only needs to happen once after all packages - not after every package: see #106.